fixtures/braid-design-system/package.json

```diff
@@ -3,10 +3,14 @@
   "private": true,
   "type": "module",
   "dependencies": {
-    "braid-design-system": "^33.1.0",
+    "braid-design-system": "^33.6.0",
```

Comment:
I would avoid updating braid here, as it's causing snapshot updates, which are just noise in this PR.

Reply:
Oh, completely missed that you converted the braid fixture to typescript, and that's why the snapshot has updated.